global-files/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,82 @@
+# Code of Conduct
+
+## Introduction
+
+Exercism is a platform centered around empathetic conversation. We have a low tolerance for communication that makes anyone feel unwelcome, unsupported, insulted or discriminated against.
+
+## Seen or experienced something uncomfortable?
+
+If you see or experience abuse, harassment, discrimination, or feel unsafe or upset, please email abuse@exercism.io. We will take your report seriously.
+
+## Enforcement
+
+We actively monitor for Code of Conduct (CoC) violations and take any reports of violations extremely seriously. We have banned contributors, mentors and users due to violations.
+
+After we receive a report of a CoC violation, we view that person's conversation history on Exercism and related communication channels and attempt to understand whether someone has deliberately broken the CoC, or accidently crossed a line. We generally reach out to the person who has been reported to discuss any concerns we have and warn them that repeated violations will result in a ban. Sometimes we decide that no violation has occurred and that no action is required and sometimes we will also ban people on a first offence. We strive to be fair, but will err on the side of protecting the culture of our community.
```

Comment:
accidently should be accidentally

Reply:
#58 fixes it. It's not worth sending out a bulk PR for that alone but it'll be synced once people update an append or the next big PR happens.

Reply:
Lol!